### DISCUSSION TITLE
I was fixing the urls with django urls

### DISCUSSION THREAD

**Discussion by anurag6569201** [4darsh-Dev/CogniGaurd#105] [General]
![image](https://github.com/4darsh-Dev/CogniGaurd/assets/140162738/b3996506-aae3-4d85-83c1-7614fb3f1955)
@4darsh-Dev when i am registering i am getting this error let me know how to remove it

**Comment by 4darsh-Dev** [comment]
@anurag6569201, acutally error is due to missing email smtp server missing credentials in .env file
When a user registers then we send an email to them to verify the email, when you're using the development server, 
you don't have access to our smtp server, 

Fix : You can add your SMTP email id and password with host and port, this will resolve the issue 

Check .env sample in readme

**Reply by anurag6569201** [reply]
ok thanks
